pkg/kube/kube_handler.go

```diff
@@ -254,6 +255,10 @@ type appRepositoryRequest struct {
 	AppRepository appRepositoryRequestDetails `json:"appRepository"`
 }
 
+type selfSubjectAccessRequest struct {
+	AppRepository appRepositoryRequestDetails `json:"appRepository"`
```

Comment:
erm, confused? Guessing this is a copy-n-paste that was left over after you found `authornizationapi.ResourceAttributes`?

Reply:
yep :) forgot to delete this